src/crons/retry-error.rb

```diff
@@ -0,0 +1,89 @@
+#!/bin/ruby
```

Comment:
This file should be moved to the utilities directory. It should never be run as a cron